compiler/optimizer/StructuralAnalysis.hpp

```diff
@@ -58,7 +58,7 @@ class TR_RegionAnalysis
 
    class StructInfo;
    typedef CS2::TableOf<StructInfo, TR::Allocator> InfoTable;
-   typedef TR::SparseBitVector SparseBitVector;
+   typedef TR::BitVector SparseBitVector;
```

Comment:
This is quite subtle.  By the name alone, someone not deeply familiar with this code would likely be expecting a sparse data structure.  Perhaps a different name should be used.